proto/README.md

```diff
@@ -14,8 +14,11 @@ custom | - | [CustomMessage](#custommessage) | SENSR additional result message (
 Field | Unit | Type | Description
 --- | --- | --- | ---
 objects | - | [Object](#object) | object list in the latest frame.
+has_objects | - | bool | A flag to check `objects` field is valid or not
```

Comment:
Out of curiosity, doesn't protobuf support validity check?

Reply:
No, basically protobuf sucks haha

> It’s impossible to differentiate a field that was missing in a protobuffer from one that was assigned to the default value.

https://reasonablypolymorphic.com/blog/protos-are-wrong/